code/modules/paperwork/desk_bell.dm

```diff
@@ -57,12 +57,13 @@
 
 /obj/item/desk_bell/proc/on_signal()
 	SIGNAL_HANDLER  // COMSIG_ASSEMBLY_PULSED
-	INVOKE_ASYNC(src, PROC_REF(try_ring))
+	INVOKE_ASYNC(src, PROC_REF(try_ring), null, TRUE) // user=null, from_signaler=TRUE (if nobody broke it)
 
-/obj/item/desk_bell/proc/try_ring(mob/user)
+/obj/item/desk_bell/proc/try_ring(mob/user, from_signaler = FALSE)
+	//WARNING: signal handler depends on user and from_signaler being the first two arguments (in that order)
```

Comment:
I don't think this comment is necessary, that's a pretty typical expectation for proc calls.
```suggestion
```